### PR TITLE
Added new test case to OfferTest

### PR DESCRIPTION
**Pull request by ashah1220**
Added a test case to check if resetState() works.

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionSha256Sum=11657af6356b7587bfb37287b5992e94a9686d5c8a0a1b60b87b9928a2decde5
```
Comment (ripcurlx) [inline gradle/wrapper/gradle-wrapper.properties:3]: Why did you try to remove the hash in this commit?

**Review by ripcurlx — CHANGES_REQUESTED** [review]
NACK - please see my comment

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (ashah1220) [reply gradle/wrapper/gradle-wrapper.properties:3]: The build fails for me unless I remove line 3 in the gradle properties.

Comment (ripcurlx) [reply gradle/wrapper/gradle-wrapper.properties:3]: Could you please post the log output in that case?

**Pull request closed without merging**